server/app/controllers/admin/ProgramAdminManagementController.java

```diff
@@ -85,6 +85,36 @@ public Result update(Http.Request request, long programId) {
     }
   }
 
+  /** Deletes an existing admin email. */
+  @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
+  public Result delete(Http.Request request, long programId, String adminEmail) {
+    try {
+      roleService.removeProgramAdmins(programId, ImmutableSet.of(adminEmail));
+      return redirect(routes.AdminProgramController.index());
```

Comment:
This will redirect to the program list. Should it redirect back to the program admin management page instead?